### PR TITLE
#22: add license file and license header to all files

### LINKED ISSUE #22: add license file and license header to all files
we need a liense file and a license header to every file 

we can adapt the ones used in pressio: see 
https://github.com/Pressio/pressio/blob/develop/LICENSE
https://github.com/Pressio/pressio/blob/develop/helper_scripts/add-sandia-license.sh

IMPORTANT: make sure Eric Parish is listed as POC for all this.

### LINKED ISSUE #26: add page to documentation that shows the license directly
we should do something like https://pressio.github.io/pressio/license.html
the page should be clearly visible in the left sidebar

### PR DESCRIPTION
**Pull request by cwschilly**
Fixes #22, fixes #26

### CODE REVIEW

File: romtools/__init__.py [hunk 1]
```diff
@@ -125,11 +125,14 @@ def getNumVars(self) -> int:
...
-
+# License
+```plaintext
+.. include:: ../LICENSE
+```
 '''
 
-__docformat__ = "markdown"  # explicitly disable rST processing in the examples above.
+__docformat__ = "restructuredtext"
+# __docformat__ = "markdown"  # explicitly disable rST processing in the examples above.
```
Comment (cwschilly) [inline romtools/__init__.py:136]: I had to enable rST in order to include the license file in the documentation. I could not see anything being affected by it--why were we explicitly disabling rST processing?

Comment (fnrizzi) [reply romtools/__init__.py:136]: becuase i followed what pdoc does in its doc, but if you say this does not change anything it is fine

File: helper_scripts/remove-sandia-license.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      if ! grep -q 'Ennio' "$file"; then
+	 echo "found file $file without a Sandia or other license. Skipping."
+	 #exit 11
+      fi
+  else
+      filetmp=$(mktemp)
+      tail -n +48 $file > $filetmp
+      mv $filetmp $file
+  fi
+done
```
Comment (fnrizzi) [inline helper_scripts/remove-sandia-license.sh:44]: did you try running this script? does it work?

File: romtools/__init__.py [hunk 3]
```diff
@@ -125,11 +125,14 @@ def getNumVars(self) -> int:
...
```
Comment (fnrizzi) [inline romtools/__init__.py:135]: please remove the markdown line and add a comment to explain why we need rst

**Comment by fnrizzi** [comment]
why is the license not shown for all the __init__.py files but only the other files? we should have the license at the top of every file inside the `romtools/` subdir

**Review by fnrizzi — CHANGES_REQUESTED** [review]
please see my comments

File: helper_scripts/remove-sandia-license.sh (hunk 2 above)
Comment (cwschilly) [reply helper_scripts/remove-sandia-license.sh:44]: No, it was shaving off too many lines. Fixed now

File: romtools/__init__.py (hunk 3 above)
Comment (cwschilly) [reply romtools/__init__.py]: Done

**Review by fnrizzi — APPROVED** [review]

**Pull request merged**